api/konfig/plugins.go

```diff
@@ -39,6 +39,9 @@ const (
 	// Injected into plugin paths when plugins are disabled.
 	// Provides a clue in flows that shouldn't happen.
 	NoPluginHomeSentinal = "/No/non-builtin/plugins!"
+
+	// Label key that indicates the resources are built from Kustomize
+	ManagedbyLabelKey = "app.kubernetes.io/managed-by"
```

Comment:
nit: this should go in `api/konfig/general.go` since it's not plugin related.

Reply:
Moved it to `api/konfig/general.go`.
Added added an env variable for `KUSTOMIZE_ENABLE_MANAGEDBY_LABEL` in this file.